### PR TITLE
FAT-16870 - MODSENDER-69 - Update the permission name.

### PR DESCRIPTION
**Pull request by gurleenkaurbp**
https://folio-org.atlassian.net/browse/FAT-16870

## Purpose
Update the mod-sender permission as part of https://folio-org.atlassian.net/browse/MODSENDER-69

## Approach
Merge at the same time as https://github.com/folio-org/mod-sender/pull/59

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Comment by gurleenkaurbp** [comment]
Hello @julianladisch ,

Any reason for the do not merge label ? I was planning to merge the PR today as I have 2 approvals.

**Pull request merged**
